modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -137,11 +138,11 @@ inline std::vector<int64_t> toORT(const cv::MatSize &sz) {
 }
 
 inline std::vector<const char*> getCharNames(const std::vector<std::string>& names) {
-    std::vector<const char*> out_vec;
-    for (const auto& el : names) {
-        out_vec.push_back(el.data());
-    }
-    return out_vec;
+    std::vector<const char*> out_ptrs;
+    out_ptrs.reserve(names.size());
+    ade::util::transform(names, std::back_inserter(out_ptrs),
+                         [](const std::string& name) { return name.data(); });
```

Comment:
I believe `c_str` is more correct than `data`

Reply:
Fixed.